engine/crates/composition/tests/composition/requires_basic/api.graphql

```diff
@@ -0,0 +1,21 @@
+type Farm {
```

Comment:
I'd consider leaving `@requires`. Contrary to `@provides`, it's not an optimization, it's more related to business logic.